### ISSUE TITLE
Secondary nav is not programmatically associated to "In this section" button

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12063]
WCAG 2.0

## Describe the defect
On VAMC pages on mobile, the secondary navigation does not have any programmatic association to the menu nor indication that it is open or closed. This will cause confusion for non-visual users are they may not know that they have successfully activated the button and that the menu items are now available.

## To Reproduce
1. Go to a VAMC page on VA.gov, like [VA Boston health care](https://www.va.gov/boston-health-care/)
2. Turn your screen reader on
3. Click or use the Enter key to activate the "In this section" secondary navigation button 
4. Note that the SR does not announce any change in context (i.e. there is not announcement that the button has been expanded, there is no announcement of a focus change, etc.)

## Additional context

1. The `aria-expanded` and `aria-controls` attribute can be used 
   1. This is used on the mobile footer sections
   2. It is a relatively simple solution

## AC / Expected behavior

- [x] I expect The aria-expanded and aria-controls attribute are used, This is used on the mobile footer sections, It is a relatively simple solution
- [x] "In this section" button must be programmatically associated to the secondary menu
- [x] Secondary navigation menu must programmatically indicate when it is open and closed
- [ ] Accessibility review 

## Screenshots
**VAMC Page - In the section, closed**
![image](https://user-images.githubusercontent.com/106678594/208992271-3c815d25-9efb-4bbe-a81c-9c8059251e64.png)

**VAMC Page - In the section, open**
![image](https://user-images.githubusercontent.com/106678594/208992341-20ce26b9-3723-4114-8f57-3c2c91981f39.png)

**Benefits Page - In the section, closed**
![image](https://user-images.githubusercontent.com/106678594/208992446-8e039b9c-d858-4904-bb63-2a4454194f80.png)

**Benefits Page - In the section, open**
![image](https://user-images.githubusercontent.com/106678594/208992487-b7c516a5-89a3-45dc-ad03-9bbab59e6219.png)

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [x] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by briandeconinck** [comment]
This issue was identified during the Collaboration Cycle Staging Review for Lovell VAMC on 2/28/2023. Applicable Collaboration Cycle labels have been added to this ticket for tracking and dashboard purposes, rather than creating a duplicate ticket.

**Comment by laflannery** [comment]
Looked at this with @ryguyk and this is approved for accessibility

**Comment by CherylEvans** [comment]
This ticket was recreated in the va.gov-team repo Ticket # [64846](https://app.zenhub.com/workspaces/collaboration-cycle-reviews-61d89e9e5f4c120011a0ad37/issues/gh/department-of-veterans-affairs/va.gov-team/64846)
